src/components/Tab/Tab.jsx

```diff
@@ -0,0 +1,9 @@
+export const Tab = ({ tab, findActive, active }) => {
+  return (
+    <li data-cy="Tab" className={tab.id === active.id ? 'is-active' : ''}>
```

Comment:
Update the condition to use `activeTabId` instead of `active.id` to match the updated prop name.